lib/src/layout_element.dart

```diff
@@ -48,14 +60,14 @@ class TableLayoutElement extends LayoutElement {
                       colWidth.substring(0, colWidth.length - 1));
                   return percentageSize != null
                       ? FlexibleTrackSize(percentageSize * 0.01)
-                      : FlexibleTrackSize(1);
+                      : IntrinsicContentTrackSize();
                 } else if (colWidth != null) {
                   final fixedPxSize = double.tryParse(colWidth);
                   return fixedPxSize != null
                       ? FixedTrackSize(fixedPxSize)
-                      : FlexibleTrackSize(1);
+                      : IntrinsicContentTrackSize();
                 } else {
-                  return FlexibleTrackSize(1);
+                  return IntrinsicContentTrackSize();
```

Comment:
Does this new flutter_layout_grid parameter correctly size when images are in a table? Curious to know whether this fixes that one issue we were having with small images inside tables.

Reply:
Yes it does!

Reply:
Awesome! I'm going to remove my comment on your image render proposal since this takes care of that.